new_platforms/vsextension/ProjectWizard/WizardImplementation.cs

```diff
@@ -33,6 +33,31 @@ public void RunFinished()
         {
         }
 
+        // Look in <folder>/conf.mk for CFG_TA_FLOAT_SUPPORT := y
+        // to see if hardware float support is enabled.
+        private bool IsHardwareFloatSupported(string folder)
+        {
+            try
+            {
+                string fileName = Path.Combine(folder, "mk\\conf.mk");
+                var lines = File.ReadLines(fileName);
+                foreach (var line in lines)
+                {
+                    if (!line.Contains("CFG_TA_FLOAT_SUPPORT"))
+                    {
+                        continue;
+                    }
+
+                    return line.TrimEnd().EndsWith("y");
+                }
+                return false;
+            }
+            catch (IOException e)
+            {
+                return false;
```

Comment:
Is it possible to write out to the Output window a message saying that the extension failed to detect hard/soft float support? The rationale being that the if processor in question does indeed support hard float but we fail to detect it, performance via soft float will be worse than it could be.

Reply:
Done